JoelFaldin/ServerSort-MuniCrud-Backend
# Testing/user service tests

# Add tests for user service

## Description 🗒️✏️

Added tests for user.service. It makes sure all services are working as expected. Also configured tests to only consider files inside of __tests__ folders.

## How to test it? 🎯⚗️🔬

- Install the dependencies using `pnpm i`.
- For prisma, run `pnpx prisma migrate dev --name init`, then run `pnpx prisma generate`. Aditionally, you can start a local page to inspect the tables using `pnpx prisma studio`, but is not necessary for the tests.
- Run `pnpm run test`, and you should see something like this:
![image](https://github.com/user-attachments/assets/6820e386-dd9a-4306-afae-5c5785a92f25)
